docs/data/data-grid/accessibility/accessibility.md

```diff
@@ -107,13 +107,20 @@ renderCell: (params) => (
 ### Navigation
 
 :::info
-The following key assignments apply to Windows and Linux users.
+The key assignments on the table below apply to Windows and Linux users.
```

Comment:
Revised this content with the intention to clarify which 'key assignments' we are referring to, as the list of key equivalences has now grown slightly larger.